pydm/widgets/shell_command.py

```diff
@@ -51,6 +51,31 @@ def __init__(self, parent=None, command=None, title=None):
         self._password = ""
         self._protected_password = ""
 
+        self.env_var = None
+
+    @Property(str)
+    def environment_variables(self):
```

Comment:
Should probably rename this and the setter to `environmentVariables` to match the style of the rest of the qt properties.

Reply:
Ok, will do!